README.md

```diff
@@ -1,29 +1,55 @@
-# Compliance Workshop Setup
+tf-compliance-workshop is a provisioner for AWS that uses Terraform for compliance workshops. It spins up one Automate server and the requisite number of workstations using the playing card system. It assumes you have terraform installed, your AWS ssh keys and secret keys, AWS CLI setup, etc. 
 
-## Things you'll need
+# Quickstart
 
- * a valid `delivery.license` file in this directory
+1. `git clone` the repo.
+1. place a valid `delivery.license` file in this directory
+1. Update .gitignore with these changes:
 
-## Running the thing
+        *.tfvars
+        *.tfstate*
+        .DS_Store
+        !example.tfvars
+        !example_secrets.tfvars
+        *.keys
+        *.license
+        *.creds
+        vendored-cookbooks/**
+        .chef/delivery-validator.pem
+        .chef/delivery-validator.pub
 
-This is all configured for the us-west region of EC2.
+1. Create a terraform.tfvars file and include your variables there. See the included example.tfvars file and make a copy of it and call it terraform.tfvars
+2. Make sure the workstations.tf file is correct for your needs. The one in this repo will make 52 workstations.
+3. Run `terraform plan`
+4. Run `terraform apply`
+5. Run ruby ./parse-state.rb terraform.tfstate
 
-Assuming I want my automate hostname prefixed with `scale` and my AWS EC2 SSH keypair is named `mykeynamehere`:
+# UPDATE THE Variables in the terraform.tfvars
+
+- aws_sshkey="your_aws_keypair_name"
+This variable refers to your AWS keypair name. If you need to setup a keypair, check out http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html
+
+- prefix="your_workshop_name"
+This variable is where you will specify the prefix, so if the prefix is "chi", the automate server FQDN will be: `chi-compliance-workshop.chefdemo.net
+
+- region - This project works only in us-west-2 at this time. The region name where your aws instances will live.
```

Comment:
Until this supports multi-region, i would not make this a configurable variable as it will end up biting someone hard. And when we do, we'll need some var maps that map region to AMI ID. I'd eliminate this for now.